code/modules/tgui/states/greyscale_menu.dm

```diff
@@ -9,6 +9,6 @@ GLOBAL_DATUM_INIT(greyscale_menu_state, /datum/ui_state/greyscale_menu_state, ne
 /datum/ui_state/greyscale_menu_state/can_use_topic(src_object, mob/user)
 	var/datum/greyscale_modify_menu/menu = src_object
 	if(!isatom(menu.target))
-		return TRUE
+		return UI_INTERACTIVE
```

Comment:
wait a sec this makes no sense when would you want to greyscale a datum

Reply:
![image](https://github.com/tgstation/tgstation/assets/58055496/be1ee5fc-98f1-4c22-bc79-6e6629d05758)
crying

Reply:
How'd you think I found this

Reply:
@Ghommie does this really make sense?

Reply:
Do whatever you want and need to fix it. It's their problem and not ours.